openassessment/templates/openassessmentblock/edit/oa_edit_basic_settings_list.html

```diff
@@ -42,21 +42,25 @@
                 </select>
             </div>
             <p class="setting-help">
-                {% trans "Specify whether learners can submit files along with their text responses. Select Images to allow JPG, GIF, or PNG files. Select PDF or Images to allow PDF files and images. Select Custom File Types to allow files with extensions that you specify below. (Use the Select Custom File Types option with caution.)" %}
+                {% trans "Specify whether learners can submit files along with their text responses." %}
             </p>
-            <div id="openassessment_submission_white_listed_file_types_wrapper" class="{% if file_upload_type != "custom" %}is--hidden{% endif %}">
+            <div id="openassessment_submission_white_listed_file_types_wrapper">
                 <div class="wrapper-comp-setting">
                     <label for="openassessment_submission_white_listed_file_types" class="setting-label">{% trans "File Types" %}</label>
                     <input id="openassessment_submission_white_listed_file_types"
                            class="input setting-input"
                            type="text"
                            value="{{ white_listed_file_types }}"
+                           {% if file_upload_type != "custom" %}disabled{% endif %}
                             />
                 </div>
                 <p class="setting-help">
                     {% trans "Enter the file extensions, separated by commas, that you want learners to be able to upload. For example: pdf,doc,docx." %}
                 </p>&nbsp;
                 <p class="setting-help message-status error"></p>
+                <p class="setting-help warning-banner extension-warning {% if file_upload_type == "custom" %}is--hidden{% endif %}">
+                    {% trans "To add more file extensions, select Custom File Types and enter the full list of acceptable file extensions you'd like to include." %}
```

Comment:
nit: I am not a fan of "you'd like" in help text. I prefer more directional comments:
`and enter the full list of acceptable file extensions to be included.

Reply:
Thanks, will run wording by Sapana 👍

Reply:
Updated 👍